src/components/ui/Merge/Vult/MergeCompleteModal.tsx

```
@@ -78,8 +83,7 @@ export const MergeCompleteModal = (props: MergeCompleteProps) => {
                 {props.amount}
               </Typography>
               <Typography size="xs" className="text_light_gray">
-                {/* US$ {Number(encodedData!.gasUsd).toFixed(2)} estimated fees */}
-                US$ 0.00 estimated fees
+                US$ {props!?.totalFees.toFixed(4)} estimated fees
```

Biome

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)